virt/virt-4-13-release-notes.adoc

```diff
@@ -118,6 +121,8 @@ Removed features are not supported in the current release.
 
 //CNV-18462 Release note: CHANGE  Retire alpha APIs for Storage
 
+//CNV-19043 Release note: CHANGE Remove rhel6 support
```

Comment:
The RN for this placeholder is on line 118; you might want to move it up or remove it, depending on how you want to handle the placeholders. :)

Reply:
I moved the placeholder up.